company_website/tests/test_view_helpers.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
+from unittest.mock import Mock
+from unittest.mock import patch
+
+from requests.exceptions import Timeout
+
+from company_website.view_helpers import generate_subresource_integrity_sha384
+
+
+def mocked_requests(url, timeout):  # pylint: disable=unused-argument
```

Comment:
Instead of adding pylint disable here please add `_` in front of both parameters thanks to this you won't get pylint warning about unused arguments